tests/resource/aws_emr_managed_scaling_policy: Managed scaling is not available for the account for this release.

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Description

The `TestAccAwsEmrManagedScalingPolicy_` acceptance tests currently fail in AWS GovCloud (US):

```
=== CONT  TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumCoreCapacityUnits
    resource_aws_emr_managed_scaling_policy_test.go:42: Step 1/2 error: Error running apply:
        Error: error putting EMR Managed Scaling Policy: ValidationException: Managed scaling is not available for the account for this release.
        	status code: 400, request id: d0355659-979a-4b1c-9293-0e7ac4d2df33


=== CONT  TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumOndemandCapacityUnits
    resource_aws_emr_managed_scaling_policy_test.go:66: Step 1/2 error: Error running apply:
        Error: error putting EMR Managed Scaling Policy: ValidationException: Managed scaling is not available for the account for this release.
        	status code: 400, request id: 3801156d-280d-4060-ad9f-f1aaf9388f83


=== CONT  TestAccAwsEmrManagedScalingPolicy_basic
    resource_aws_emr_managed_scaling_policy_test.go:18: Step 1/2 error: Error running apply:
        Error: error putting EMR Managed Scaling Policy: ValidationException: Managed scaling is not available for the account for this release.
        	status code: 400, request id: 9324e1bf-eae7-4676-9b65-26e47759f2f8


=== CONT  TestAccAwsEmrManagedScalingPolicy_disappears
    resource_aws_emr_managed_scaling_policy_test.go:90: Step 1/1 error: Error running apply:
        Error: error putting EMR Managed Scaling Policy: ValidationException: Managed scaling is not available for the account for this release.
        	status code: 400, request id: bb061a65-c104-41f3-ab27-a6eb992daa57


--- FAIL: TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumOndemandCapacityUnits (421.03s)
--- FAIL: TestAccAwsEmrManagedScalingPolicy_disappears (445.83s)
--- FAIL: TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumCoreCapacityUnits (462.71s)
--- FAIL: TestAccAwsEmrManagedScalingPolicy_basic (466.76s)
```

These tests likely need an additional `PreCheck` function to skip the tests in unsupported regions.

### Affected Resource(s)

* aws_emr_managed_scaling_policy

--- a/aws/resource_aws_emr_managed_scaling_policy_test.go
+++ b/aws/resource_aws_emr_managed_scaling_policy_test.go
@@ -17,6 +17,7 @@ func TestAccAwsEmrManagedScalingPolicy_basic(t *testing.T) {
 	rName := acctest.RandomWithPrefix("tf-acc-test")
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheckSkipEmrManagedScalingPolicy(t),
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSEmrManagedScalingPolicyDestroy,
 
@@ -41,6 +42,7 @@ func TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumCoreCapacityUnits(t 
 	rName := acctest.RandomWithPrefix("tf-acc-test")
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheckSkipEmrManagedScalingPolicy(t),
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSEmrManagedScalingPolicyDestroy,
 
@@ -65,6 +67,7 @@ func TestAccAwsEmrManagedScalingPolicy_ComputeLimits_MaximumOndemandCapacityUnit
 	rName := acctest.RandomWithPrefix("tf-acc-test")
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheckSkipEmrManagedScalingPolicy(t),
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSEmrManagedScalingPolicyDestroy,
 
@@ -89,6 +92,7 @@ func TestAccAwsEmrManagedScalingPolicy_disappears(t *testing.T) {
 	rName := acctest.RandomWithPrefix("tf-acc-test")
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheckSkipEmrManagedScalingPolicy(t),
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckAWSEmrManagedScalingPolicyDestroy,
 		Steps: []resource.TestStep{
@@ -102,6 +106,13 @@ func TestAccAwsEmrManagedScalingPolicy_disappears(t *testing.T) {
 			},
 		},
 	})
+}
+
+// testAccErrorCheckSkipEmrManagedScalingPolicy skips tests that have error messages indicating unsupported features
+func testAccErrorCheckSkipEmrManagedScalingPolicy(t *testing.T) resource.ErrorCheckFunc {
+	return testAccErrorCheckSkipMessagesContaining(t,
+		"Managed scaling is not available",
+	)
 }
 
 func testAccAWSEmrManagedScalingPolicy_basic(r string) string {

